### ISSUE TITLE
Primefaces mobile footer does not stay fixed

### ISSUE THREAD

**Issue by basiel007** [primefaces/primefaces#410]
In PrimeFaces 5.2 the footer in primefaces mobile does not stay at bottom fixed when updating a dataList after an ajax request. 
The sample code is below just click on any selectonebutton in the table to see the effect.

Here the code sample:

```
<?xml version='1.0' encoding='UTF-8' ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:h="http://xmlns.jcp.org/jsf/html"
      xmlns:f="http://xmlns.jcp.org/jsf/core"
      xmlns:pm="http://primefaces.org/mobile"
      xmlns:p="http://primefaces.org/ui">
    <f:view renderKitId="PRIMEFACES_MOBILE" />
    <h:head>
        <title>Facelet Title</title>
    </h:head>
    <h:body>
        <h:form>
            <pm:content>
                <h:outputLabel value="test content" />

                <p:dataList id="datalist" var="currentComponent" value="#{homePageController.persons}">
                    <h:panelGrid columns="2">
                        <h:outputText value="#{currentComponent.firstName}"/>
                        <p:selectOneButton>
                            <f:selectItem itemLabel="On"/>
                            <f:selectItem itemLabel="Off"/>
                            <p:ajax update="datalist"/>
                        </p:selectOneButton>
                    </h:panelGrid>

                </p:dataList>


            </pm:content>
            <pm:footer  fixed="true">
                <p:tabMenu >
                    <p:menuitem value="tab1"/>
                    <p:menuitem value="tab2"/>
                </p:tabMenu>
            </pm:footer>
        </h:form>

    </h:body>
</html>
```

HomePageController.java:

```
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package be.fgov.crbs.testmanagehome;

import java.util.ArrayList;
import java.util.List;
import javax.annotation.PostConstruct;
import javax.faces.bean.ManagedBean;
import javax.faces.bean.ViewScoped;

/**
 *
 * @author basiel007
 */
@ManagedBean
@ViewScoped
public class HomePageController {

    /**
     * Creates a new instance of HomePageController
     */
    public HomePageController() {
    }

    @PostConstruct
    private void initController() {
        persons = new ArrayList<PersonComponent>();
        persons.add(new PersonComponent("a", "aa"));
        persons.add(new PersonComponent("b", "bb"));
        persons.add(new PersonComponent("d", "dd"));
        persons.add(new PersonComponent("c", "cc"));
        persons.add(new PersonComponent("e", "ee"));
        persons.add(new PersonComponent("f", "ff"));
        persons.add(new PersonComponent("g", "gg"));
        persons.add(new PersonComponent("h", "hh"));
        persons.add(new PersonComponent("i", "ii"));
        persons.add(new PersonComponent("j", "jj"));
        persons.add(new PersonComponent("k", "kk"));
        persons.add(new PersonComponent("l", "ll"));
        persons.add(new PersonComponent("m", "mm"));
        persons.add(new PersonComponent("n", "nn"));
        persons.add(new PersonComponent("o", "oo"));
        persons.add(new PersonComponent("p", "pp"));
        persons.add(new PersonComponent("q", "qq"));
        persons.add(new PersonComponent("r", "rr"));
        persons.add(new PersonComponent("s", "ss"));
        persons.add(new PersonComponent("t", "tt"));

    }

    private List<PersonComponent> persons;

    public List<PersonComponent> getPersons() {
        return persons;
    }

    public void setPersons(List<PersonComponent> persons) {
        this.persons = persons;
    }

}
```

PersonComponent.java

```
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package be.fgov.crbs.testmanagehome;

import java.io.Serializable;

/**
 *
 * @author basiel007
 */
public class PersonComponent implements Serializable {

    private String name;
    private String firstName;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public PersonComponent() {
    }

    public PersonComponent(String name, String firstName) {
        this.name = name;
        this.firstName = firstName;
    }

}
```

**Comment by kukel** [comment]
what if you put the `pm:footer` outside the form and the form inside the `pm:content`

**Comment by basiel007** [comment]
Same problem (i also tried putting the tabmenu inside content and outside form of table) also same problem with code below:

```
<?xml version='1.0' encoding='UTF-8' ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:h="http://xmlns.jcp.org/jsf/html"
      xmlns:f="http://xmlns.jcp.org/jsf/core"
      xmlns:pm="http://primefaces.org/mobile"
      xmlns:p="http://primefaces.org/ui">
    <f:view renderKitId="PRIMEFACES_MOBILE" />
    <h:head>
        <title>Facelet Title</title>
    </h:head>
    <h:body>

        <pm:content>
            <h:form>
                <h:outputLabel value="test content" />

                <p:dataList id="datalist" var="currentComponent" value="#{homePageController.persons}">
                    <h:panelGrid columns="2">
                        <h:outputText value="#{currentComponent.firstName}"/>
                        <p:selectOneButton>
                            <f:selectItem itemLabel="On"/>
                            <f:selectItem itemLabel="Off"/>
                            <p:ajax update="datalist"/>
                        </p:selectOneButton>
                    </h:panelGrid>

                </p:dataList>


            </h:form>

            <pm:footer  fixed="true">
                <h:form>
                    <p:tabMenu >
                        <p:menuitem value="tab1"/>
                        <p:menuitem value="tab2"/>
                    </p:tabMenu>
                </h:form>
            </pm:footer>
        </pm:content>


    </h:body>
</html>
```

**Comment by tandraschko** [comment]
PrimeFaces Mobile will be deprecated  in 6.2 and removed in 6.3 ( see https://www.primefaces.org/primefaces-6-2-roadmap/)
Therefor i will close this issue as won't fix. If you will provide a PR to fix this issue until 6.2 RC1, i can merge it.
